webpack.mix.js

```diff
@@ -15,6 +15,14 @@ const tailwindcss = require('tailwindcss')
  |
  */
 
+// Provide inline source maps.
+if ( ! mix.inProduction()) {
+  mix.webpackConfig({
+    devtool: 'inline-source-map',
```

Comment:
Curious, why is this needed? Is `sourceMaps()` not enough?

Reply:
I'm afraid not, it's an known issue.
https://github.com/JeffreyWay/laravel-mix/issues/1793

Reply:
Also I didn't use `devtool: 'source-map'`. You likely don't want the extra files ending up in `/public` and getting accidentally committed.

Reply:
Are you sure? I just tried with only `.sourceMaps()`, and it seemed to work fine. Or am I missing something?

![image](https://user-images.githubusercontent.com/882133/56507806-7cc93300-64f0-11e9-8001-f1d9a05df4a1.png)